.github/workflows/release.yaml

```diff
@@ -40,3 +40,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - run: gh workflow run ci.yml --repo nextstrain/docker-base
+  create-github-release:
+    needs: [run]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.version}}
+    - name: Create GitHub Release
+      run: |
+        gh release create \
+          "${{ github.event.inputs.version}}" \
+          --repo nextstrain/augur \
+          --title "${{ github.event.inputs.version}}" \
+          --target "$(git rev-list -1 ${{ github.event.inputs.version}})" \
```

Comment:
I think this isn't quite the right `git` command... though it depends what `gh release create` wants.

`git rev-list -1` will output the id of the `commit` object associated with the tag name in `github.event.inputs.version`. This is different than the id of the `tag` object itself. `git rev-parse` is more appropriate here for parsing a symbolic ref into a raw id. Specifically, I think something like

    git rev-parse --verify '${{github.event.inputs.version}}^{tag}'

would be more what we want. `--verify` has some nice semantics which match our requirements here and the `…^{tag}` operator ensures that we're dereferencing a tag not some other object.

I suspect `gh release create`/we want the actual tag object not the commit object the tag points to. That said, can we just pass `${{github.events.inputs.version}}` directly to `--target` and let `gh release create` resolve it as necessary?

Reply:
Ahh, I see the test run in your fork now where it does appear `gh release create` figured it out from the commit object passed to `--target`. So I guess it's just my own curiosity now. :-)

Though even still, using `rev-parse --verify '${{github.event.release.version}'` feels more of a match for what we want than `rev-list -1`.

But consider this a non-blocking comment now.

Reply:
Updated with a new script `devel/create-github-release` ported over from https://github.com/nextstrain/cli/pull/186. It uses `rev-parse --verify`.